### ISSUE TITLE
Suggestion to create more models packs

### ISSUE THREAD

**Issue by tomas122727** [hydrusbeta/hay_say_ui#11]
hello you might find this useful to add more model packs by using a website called weights.gg 
https://www.weights.gg/
search terms on the sidepanel
"mlp" 
"MLP" 
"My Little Pony" 
it will find u heaps of RVC MODELS of many different ponys

**Comment by hydrusbeta** [comment]
Thanks for the suggestion! There are definitely pony models on that website that I was not aware of. I'll scour through it sometime and at least add all the characters that aren't in Hay Say already.

**Comment by tomas122727** [comment]
yes its a humongous list also always check it cause that list is set to grow
